events/events.go

```diff
@@ -361,6 +366,41 @@ type OperationDeprovisionedData struct {
 	Team   *Team        `json:"team,omitempty"`
 }
 
+// OperationResized represents a resize operation event.
+type OperationResized struct {
+	BaseBody
+	Data OperationResizedData `json:"data"`
+}
+
+// OperationResizedData holds the event specific data.
+type OperationResizedData struct {
+	OperationID manifold.ID `json:"operation_id"`
+
+	ResourceID manifold.ID `json:"resource_id"`
+	Resource   *Resource   `json:"resource,omitempty"`
+
+	UserID *manifold.ID `json:"user_id,omitempty"`
+	User   *User        `json:"user,omitempty"`
+
+	TeamID *manifold.ID `json:"team_id,omitempty"`
+	Team   *Team        `json:"team,omitempty"`
+
+	ProjectID *manifold.ID `json:"project_id,omitempty"`
+	Project   *Project     `json:"project,omitempty"`
+
+	ProviderID *manifold.ID `json:"provider_id,omitempty"`
+	Provider   *Provider    `json:"provider,omitempty"`
+
+	ProductID *manifold.ID `json:"product_id,omitempty"`
+	Product   *Product     `json:"product,omitempty"`
+
+	PlanID *manifold.ID `json:"plan_id,omitempty"`
```

Comment:
Why is this optional?
And should we have `OldPlanID` & `NewPlanID`?

Reply:
Yeah. That's a really good point. I will change to have both.

Reply:
:+1: Cool I'll approve once you update.